### PR TITLE
[DOCS] Mention that ESF may result in additional charges

### LINKED ISSUE #444: [Docs] Review and improve guidance around AWS costs
**Describe the enhancement:**
We've had feedback from one customer that the documentation could have highlighted potential AWS-related costs more clearly. This was initially understood to be a functional issue that has since been rectified but it makes sense to review the current guidance (e.g. there is a troubleshooting topic that could be mirrored as a high profile warning on index page) with a view to improving and making more prominent, especially for older versions.

Suggested contacts:
Tech lead: @tommyers-elastic 
Product lead: @vinaychandrasekhar
Writer: @dedemorton 
Escalations: Basile Martin

This might be duplicate issue but waiting on confirmation and we can always link and dedupe...

### PR DESCRIPTION
**Pull request by dedemorton**
## What does this PR do?
Adds a statement to the overview page in the ESF docs:

image

Note that I decided not to put this statement on the index page most users don't see the index page (google sends them to the overview).

## Why is it important?

Users need to know in advance if they are doing something that might result in additional charges.

## Checklist

- [x] My code follows the style guidelines of this project

## Related issues

- Closes #444

### CODE REVIEW

**Comment by dedemorton** [comment]
@ollyhowell Is this what you had in mind?
@tommyers-elastic Do we need to say more on the troubleshooting page?

**Comment by kaiyan-sheng** [comment]
https://github.com/elastic/elastic-serverless-forwarder/pull/449 is merged so this PR should pass the CI now. Thanks!

File: docs/en/aws-elastic-serverless-forwarder.asciidoc [hunk 1]
```diff
@@ -13,6 +13,10 @@ The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda functi
 
 The Elastic Serverless Forwarder works with {stack} 7.16 and later.
 
+IMPORTANT: Using Elastic Serverless Forwarder may result in additional charges
+on your next cloud provider billing statement due to increased usage. To learn
+how to minimize additional charges, refer to <<preventing-unexpected-costs>>.
```
Comment (kaiyan-sheng) [inline docs/en/aws-elastic-serverless-forwarder.asciidoc:18]: How about simplifying it to this:
```suggestion
IMPORTANT: Using Elastic Serverless Forwarder may result in additional charges. To learn
how to minimize additional charges, refer to <<preventing-unexpected-costs>>.
```

File: docs/en/aws-elastic-serverless-forwarder.asciidoc [hunk 2]
```diff
@@ -13,6 +13,11 @@ The Elastic Serverless Forwarder is an Amazon Web Services ({aws}) Lambda functi
 
 The Elastic Serverless Forwarder works with {stack} 7.16 and later.
 
+IMPORTANT: Using Elastic Serverless Forwarder may result in additional charges
+on your next cloud provider billing statement due to increased usage. To learn
```
Comment (dedemorton) [inline docs/en/aws-elastic-serverless-forwarder.asciidoc:17]:
```suggestion
```

**Review by kaiyan-sheng — APPROVED** [review]

**Comment by ollyhowell** [comment]
> @ollyhowell Is this what you had in mind? @tommyers-elastic Do we need to say more on the troubleshooting page?

This is great - thanks for picking this up and solving so quickly @dedemorton !

**Pull request merged**